src/Error/ErrorLogger.php

```diff
@@ -85,6 +84,31 @@ public function logError(PhpError $error, ?ServerRequestInterface $request = nul
         $this->log($level, $message);
     }
 
+    /**
+     * Generate the message for the error
+     *
+     * @param PhpError $error The exception to log a message for.
+     * @param bool $includeTrace Whether or not to include a stack trace.
+     * @return string Error message
+     */
+    private function getErrorMessage(PhpError $error, bool $includeTrace = false): string
```

Comment:
It might be better to move this to 5.next and make this protected instead.
But for me either way works for now.

Reply:
Change from private to protected was planned, but forgotten :D

Reply:
But then it must go into 5.next with a note in the migration guide IMO.